### PR TITLE
all: Add github workflows for linting and test, fix lint issues

### PR DESCRIPTION
**Pull request by matoous**
Add github worfklows for linting, tests and benchmarking.
Fix all issues reported by golangci-lint.

This test will be automatically run on every PR so we can check the
results of golangci-lint, tests and more right in the github
web.

### CODE REVIEW

File: bootstrap/bootstrap.go [hunk 1]
```diff
@@ -164,22 +163,24 @@ func (g *Generator) Run() error {
 		defer os.Remove(f.Name()) // will not remove after rename
 	}
 
+	//nolint:gosec
```
Comment (mrVanboy) [inline bootstrap/bootstrap.go]: could we plz also write additional comment about why we are introducing `//nolint` directive?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+on: [push]
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.13
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13.3
```
Comment (mrVanboy) [inline .github/workflows/test.yml]: could we also add a separate stage to test if the code works with 1.12 version?

**Review by mrVanboy — CHANGES_REQUESTED** [review]

File: bootstrap/bootstrap.go (hunk 1 above)
Comment (matoous) [reply bootstrap/bootstrap.go]: Added explanatory comments to all `//nolint` directives.

File: .github/workflows/test.yml (hunk 2 above)
Comment (matoous) [reply .github/workflows/test.yml]: Added [build matrix](https://help.github.com/en/github/automating-your-workflow-with-github-actions/configuring-a-workflow#configuring-a-build-matrix) that runs test on multiple versions of golang. See the result: https://github.com/kiwicom/easycql/runs/281504032

**Review by mrVanboy — APPROVED** [review]
Looks nice 👍

@martin-sucha merge it if you think that this PR is fine.

**Pull request merged**